boards/FSM/Src/main.c

```diff
@@ -609,12 +610,12 @@ static void MX_TIM2_Init(void)
 
     /* USER CODE END TIM2_Init 1 */
     htim2.Instance               = TIM2;
-    htim2.Init.Prescaler         = 0;
-    htim2.Init.CounterMode       = TIM_COUNTERMODE_DOWN;
+    htim2.Init.Prescaler         = 1;
+    htim2.Init.CounterMode       = TIM_COUNTERMODE_UP;
```

Comment:
reminder to make one of them count down
![image](https://user-images.githubusercontent.com/32230980/156908207-0de7c322-43ed-4976-8b6d-77cd92e616f7.png)